aten/src/ATen/native/native_functions.yaml

```diff
@@ -1051,6 +1051,9 @@
 - func: _copy_from(Tensor self, Tensor dst, bool non_blocking=False) -> Tensor
   dispatch: {}
 
+- func: _copy_from_temp(Tensor self, Tensor dst) -> Tensor
```

Comment:
comment

Reply:
yep. It needs a better name too 😛 I originally put `_copy_from_out`, but the codegen then flags it as an `out` function.

From what I can tell, the only different between the two is `_copy_from` will fail if you pass in tensors of different sizes, while `_copy_from_temp` will resize correctly and also null out the views on the XLA tensor ([here](https://github.com/pytorch/xla/blob/f41a53458d12b19027f02e5ec34f51d187a63368/torch_xla/csrc/tensor.cpp#L882)). So maybe `_copy_from_and_resize`.